client/src/components/UserPanel.js

```diff
@@ -66,7 +66,7 @@ function UserPanel(props) {
         <Typography style={{ fontWeight: "bold" }}>{location}</Typography>
       </Grid>
       <Grid item id="edit" className={classes.item}>
-        <UserEditDialog location={location} description={description} />
+        <UserEditDialog id={props.id} />
```

Comment:
The location and description are passed in so when the edit dialog pops up, the original location and description are in the textfield already and can be modified